k8s/logs-spack-io/deployments.yaml

```diff
@@ -0,0 +1,352 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: logs-spack-io-redirect
+  namespace: logs-spack-io
+  labels:
+    app: logs-spack-io
+    svc: web-redirect
+    environment: matomo
+
+spec:
+  selector:
+    matchLabels:
+      app: logs-spack-io
+      svc: web-redirect
+      environment: matomo
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: logs-spack-io
+        svc: web-redirect
+        environment: matomo
+    spec:
+      containers:
+      - name: matomo-nginx
+        imagePullPolicy: Always
+        image: "nginx:alpine"
+        command: ["nginx", "-g", "daemon off;", "-c", "/config/nginx.conf"]
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 500m
+            memory: 512Gi
+        ports:
+          - name: http
+            containerPort: 80
+            protocol: TCP
+        volumeMounts:
+          - mountPath: /config/nginx.conf
+            name: config
+            subPath: nginx.conf
+          - mountPath: /var/www/html
+            name: html-files
+        env:
+        - name: MOUNT_PATH
+          value: "/data"
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: spack-aws-credentials
+              key: aws_access_key_id
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: spack-aws-credentials
+              key: aws_secret_access_key        
```

Comment:
For AWS operations, we've switched to using kubernetes service accounts with an attached IAM Role.  I'll sync with you out-of-band to work this part out.